jest.config.js

```diff
@@ -92,7 +92,8 @@ module.exports = {
   moduleNameMapper: {
     'c3': '<rootDir>/__mocks__/c3.js',
     '\\.(css|less|sass|scss)$': '<rootDir>/spec/javascripts/__mocks__/styleMock.js',
-    '\\.(gif|ttf|eot)$': '<rootDir>/spec/javascripts/__mocks__/fileMock.js'
+    '\\.(gif|ttf|eot)$': '<rootDir>/spec/javascripts/__mocks__/fileMock.js',
+    'nanoid': '<rootDir>/node_modules/nanoid/index.browser.cjs'
```

Comment:
I don't know what is this and why it works, but it solved this Jest problem: https://app.circleci.com/pipelines/github/3scale/porta/30520/workflows/820ae665-4abd-41aa-a035-1127ff99bba8/jobs/342079

I took the solution from here: https://github.com/rjsf-team/react-jsonschema-form/issues/3484#issuecomment-1461524013